README.md

```diff
@@ -85,9 +92,12 @@ Install from your local checked-out copy as an "editable" install:
 If you want to run the unit or regression tests and/or build the docs, you can make sure those dependencies are
 installed too:
 
-    pip install -e .[test]
-    pip install -e .[docs]
-    pip install -e .[test,docs]
+    pip install -e ".[test]"
+    pip install -e ".[docs]"
+    pip install -e ".[test,docs]"
+
+> **Important**\
```

Comment:
I don't think we need this. The only reason I referenced this document is because roman_datamodels does not have an install procedure.